Today/Controllers/ReminderDetailViewController/ReminderDetailDataSource.swift

```diff
@@ -18,7 +18,7 @@ class ReminderDetailDataSource: NSObject {
     
     // MARK: - Static Properties
     
-    static let reminderDetailCellIdentifier = "ReminderDetailCell"
+    static let reminderDetailCellIdentifier = Identifiers.reminderDetailCellIdentifier
```

Comment:
Its not required when you are using the method in UIViewController extension to initialize UIViewController.

Reply:
This property is for the cell's identifier, and the method is for the entire view controller.
Its used here...
![image](https://user-images.githubusercontent.com/85882612/130217029-74762bf1-0e74-4d27-8952-1f052080f9f5.png)